tests/_cur_dir.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
please add the Fuzion header like in other scripts

Reply:
@fridis done